node/config/types.go

```diff
@@ -538,19 +538,13 @@ type FeeConfig struct {
 }
 
 type FevmConfig struct {
-	// EnableEthRPC enables eth_ rpc, and enables storing a mapping of eth transaction hashes to filecoin message Cids.
-	// This will also enable the RealTimeFilterAPI and HistoricFilterAPI by default, but they can be disabled by config options above.
+	// EnableEthRPC enables eth_ RPC methods and will also enable the RealTimeFilterAPI.
```

Comment:
We need to remove all references to `RealTimeFilterAPI` an `HistoricFilterAPI`:

* 2 in this file and what's built from it: node/config/doc_gen.go, documentation/en/default-lotus-config.toml
* 1 itests/eth_config_test.go

Also `HistoricFilterAPI`:

* 1 in this file and what's built from it: node/config/doc_gen.go, documentation/en/default-lotus-config.toml

Reply:
Done.